doc/api/buffer.md

```diff
@@ -14,7 +14,7 @@ into the JavaScript language. A number of additional methods are supported
 that cover additional use cases. Node.js APIs accept plain [`Uint8Array`][]s
 wherever `Buffer`s are supported as well.
 
-Instances of the `Buffer` class, and [`Uint8Array`][]s in general,
+Instances of the `Buffer` class and [`Uint8Array`][], in general,
```

Comment:
Welcome, @tomanagle, and thanks for the pull request.

Moving the comma is incorrect.  The words _in general_ refer to `Uint8Array` instances in general, not `Buffer`.

The floating `s` doesn't bother me much, but I would be 👍 on a change to get rid of it anyway. So if you want to restore the comma placement, I think we're good:

```suggestion
Instances of `Buffer`, and instances of [`Uint8Array`][] in general,
```